ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/includes/status.jsp

```diff
@@ -0,0 +1,34 @@
+<%--
+  ~ #%L
+  ~ ACS AEM Commons Bundle
+  ~ %%
+  ~ Copyright (C) 2013 Adobe
+  ~ %%
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~ #L%
+  --%>
+
+<%-- Running --%>
+<section class="coral-Well" ng-show="app.running">
+
+    <div id="results" class="notification hidden">
+        <div id="replication-agents-info">
+            <h2>Your replication request has been submitted.</h2>
+            <p>This request may take some time to complete. Please review the Replication agent logs and
+                configurations the complete replication status does not appear below shortly.</p>
+            <div class="message"></div>
+        </div>
+        <div id="replication-queue-message"></div>
+    </div>
+
+</section>
```

Comment:
Does this need to be a JSP? It contains only HTML markup.

Reply:
Fixed. Please check